hopsworks-common/src/main/java/io/hops/hopsworks/common/security/OpensslOperations.java

```diff
@@ -258,7 +264,10 @@ public void validateCertificate(X509Certificate certificate) throws IOException
       commands.add(OPENSSL);
       commands.add("verify");
       commands.add("-CAfile");
-      commands.add(Paths.get(settings.getIntermediateCaDir(), "certs", "ca-chain.cert.pem").toString());
+      commands.add(Paths.get(pki.getCACertsDir(caType), "ca-chain.cert.pem").toString());
```

Comment:
This should be a method in PKI. Depending on the intermediate CA the chain file name "ca-chain.cert.pem" might differ